lib/util.js

```diff
@@ -229,6 +230,18 @@ function callbackify(original) {
   return callbackified;
 }
 
+/**
+ * @param {number} err
+ * @returns {string}
+ */
+function getSystemErrorMessage(err) {
+  validateNumber(err, 'err');
```

Comment:
Why do we need to call validateNumber since in the next line we check for SafeNumber?

Reply:
Same as name getter

https://github.com/juanarbol/node/blob/cf6d05950d991b0cbeb06aa10d5bcf79c485c81f/lib/util.js#L250

This more about the current codebase.